sharktank/tests/models/punet/sharded_resnet_block_with_iree_test.py

```diff
@@ -6,273 +6,127 @@
 
 import pytest
 import sys
-
+import iree.runtime
```

Comment:
Could we cleanup the import order a bit? I usually try to do alphabetical order with `from` and `import` separated out, e.g.

```
from pathlib import Path
from typing import List

import iree.compiler
import iree.runtime
import pytest
import sys
import torch
```

etc

Reply:
Done.